docs/create-a-release.md

```diff
@@ -0,0 +1,57 @@
+# Creating a new release
```

Comment:
@kuronosec  lets link to this doc from the main readme file so it is easier to find

Reply:
perfect, done!